doc/source/dev/contributor/quickstart_ubuntu.rst

```diff
@@ -0,0 +1,113 @@
+:orphan:
+
+.. _quickstart-ubuntu:
+
+=================================================
+Development Environment Quickstart Guide (Ubuntu)
+=================================================
+
+This quickstart guide will cover:
+
+* setting up and maintaining a development environment, including installing compilers and SciPy build dependencies;
+* creating a personal fork of the SciPy repository on GitHub;
+* using git to manage a local repository with development branches;
+* performing an in-place build of SciPy; and
+* creating a virtual environment that adds this development version of SciPy to the Python path
+
+in Ubuntu.
+
+.. note::
+
+	This guide does not present the only way to set up a development environment; there are many valid choices of Python distribution, C/Fortran compiler, and installation options. The steps here can often be adapted for other choices, but we cannot provide documentation tailored for them.
+
+	This guide assumes that you are starting without an existing Python 3 installation. If you already have Python 3, you might want to uninstall it first to avoid ambiguity over which Python version is being used at the command line.
+
+.. _quickstart-ubuntu-build:
+
+Building SciPy
+--------------
+
+*Consider watching the companion video* `Anaconda SciPy Dev: Part I (macOS)`_ *before starting. Even though the video is intended for macOS users, most of the procedures are the same or very similar.*
+
+#. Download, install, and test the latest release of the `Anaconda Distribution of Python`_. In addition to the latest version of Python 3, the Anaconda distribution includes dozens of the most popular Python packages for scientific computing, the Spyder integrated development environment (IDE), the ``conda`` package manager, and tools for managing virtual environments.
+
+   In Ubuntu, the Anaconda download comes as a ``.sh`` file. `In a terminal, navigate <https://help.ubuntu.com/community/UsingTheTerminal>`_ to the directory in which the ``.sh`` file is located and enter ``./file.sh``, where ``file.sh`` is to be replaced with the full name of the Anaconda download. This starts the installation process; from there, simply follow the prompts.
+
+#. Install git. An easy way to do this is to enter the command ``sudo apt-get install git`` in the terminal and follow the prompts. We'll use this software to download and manage the SciPy source code.
```

Comment:
``apt`` will be sufficient rather than the whole ``apt-get``. From the apt man page:

> apt provides a high-level commandline interface for the package management system. It is intended as an end user interface and enables some options better suited for interactive usage by default compared to more specialized APT tools like apt-get(8) and apt-cache(8).

Although I admit this is a *very* minor.

Reply:
Done.